### PR TITLE
fix(various): Small formatting and language fixes

### PR DESCRIPTION
**Pull request by lobsterkatie**
This is a bunch of small fixes and tweaks extracted from my work adding `beforeSendTransaction` to the docs - autoformatting, proofreading, and the occasional clarification. 

Note to reviewers: Each commit is atomic and accurately described, so it may be easier to review just going through them that way.

### CODE REVIEW

File: src/platforms/javascript/common/configuration/integrations/plugin.mdx [hunk 1]
```diff
@@ -42,7 +42,9 @@ This integration captures all `Console API` calls and redirects them to Sentry u
 
 _Import name: `Sentry.Integrations.Debug`_
 
-This integration allows you to inspect the content of the processed event that will be passed to `beforeSend`, and effectively send it to the Sentry SDK. It will *always* run as the last integration, no matter when it was registered.
+This integration allows you to inspect the contents of the processed event and `hint` object that will be passed to `beforeSend`. It will _always_ run as the last integration, no matter when it was registered.
+
+Note that this is different than setting `debug: true` in your `Sentry.init` options, which will enable debug logging in the console.
```
Comment (lizokm) [inline src/platforms/javascript/common/configuration/integrations/plugin.mdx:47]:
```suggestion
Note, that this is different than setting `debug: true` in your `Sentry.init` options, which will enable debug logging in the console.
```

File: src/platforms/javascript/common/troubleshooting/index.mdx [hunk 2]
```diff
@@ -346,7 +356,7 @@ const client1 = new Sentry.BrowserClient({
   integrations: [...Sentry.defaultIntegrations, new HappyIntegration()],
   beforeSend(event) {
     console.log("client 1", event);
-    return null; // Returning null does not send the event
+    return null; // Returning `null` causes the event not to be sent
```
Comment (lizokm) [inline src/platforms/javascript/common/troubleshooting/index.mdx:359]:
```suggestion
    return null; // Returning `null` causes the event to not be sent
```

**Review by lizokm — APPROVED** [review]
Thanks so much for this awesome cleanup effort!

File: src/docs/product/relay/index.mdx [hunk 3]
```diff
@@ -32,23 +32,15 @@ Sentry already scrubs PII in two places:
...
-  configure data scrubbing in a centralized place, configure your SDK to send events to Relay. Relay uses the privacy settings configured in Sentry, and scrubs PII before forwarding data to Sentry.
+- If you cannot send PII outside your infrastructure, but you still prefer to configure data scrubbing in a centralized place, configure your SDK to send events to Relay. Relay uses the privacy settings configured in Sentry, and scrubs PII before forwarding data to Sentry.
 
-- If you must enforce strict data privacy requirements, you can configure SDKs to
-  scrub PII using the [`before-send` hooks](/platform-redirect/?next=/configuration/options/%23hooks), which prevents data from being
-  collected on the device. This may require you to replicate the same logic across
-  your applications, and may impact performance.
+- If you must enforce strict data privacy requirements, you can configure SDKs to scrub PII using the [`beforeSend` hook](/platform-redirect/?next=/configuration/options/%23hooks), which prevents data from being collected on the device. This may require you to replicate the same logic across your applications, and may impact performance.
```
[the remark this answers is not in the document]
Comment (lobsterkatie) [reply src/docs/product/relay/index.mdx:43]: This isn't causal - both things are consequences of scrubbing PII in `beforeSend`.

**Pull request merged**